iotdb-cli/src/main/java/org/apache/iotdb/cli/tool/ImportCsv.java

```diff
@@ -283,27 +286,27 @@ private static void loadDataFromCSV(File file, int index) {
       StringBuilder sbd = new StringBuilder();
       ArrayList<Integer> colIndex = entry.getValue();
       sbd.append("insert into " + entry.getKey() + "(timestamp");
-      int skipcount = 0;
+      int skipCount = 0;
       for (int j = 0; j < colIndex.size(); ++j) {
-        if (data[entry.getValue().get(j) + 1].equals("")) {
-          skipcount++;
+        if ("".equals(data[entry.getValue().get(j) + 1])) {
```

Comment:
I think it's a little bit informal, and it's better to be {data[entry.getValue().get(j) + 1].isEmpty()}